react/features/conference/components/native/carmode/microphone-button/index.js

```diff
@@ -0,0 +1,3 @@
+// @flow
```

Comment:
Please don't create a directory for each button, and don't create all these index files.